openstack/grafana/templates/_pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.persistence.enabled }}
```

Comment:
Note: this file begins with _ so is not deployed, Please ignore contents for now.